Day-I-Social-Card/social-card/src/DB/pokemon.js

```diff
@@ -1,9 +1,9 @@
 const pokemon = {
-    userName: 'Ash',
+    userName: 'Ash Ketchum',
     profileIcon: 'https://orig00.deviantart.net/1d83/f/2013/104/9/a/profile_picture_by_askrp_ash_ketchum-d61rak5.jpg',
```

Comment:
Your profile Icons start to become difficult to see once the window reaches about 900px wide. Responsiveness of the cards is 🔥 though. Great job pushing yourself to add different features.